actions/build_docs/build_docs.sh

```diff
@@ -37,8 +37,13 @@ ${cmake_command} -H. -Bbuild -DBUILD_DOCS=ON -DONLY_BUILD_DOCS=ON
 ${cmake_command} --build build --target "${doc_target}"
 
 # Step 2: Migrate the Doxygen documentation to the docs source
-mkdir docs/build
-mkdir docs/build/html
+# mkdir docs/build
+# mkdir docs/build/html
+
+cd build
+mkdir html
+cd ..
```

Comment:
Is this right? 

This looks like you're making the `html` directory and then moving the empty directory to the documentation location. When you run Doxygen, the `build/html` directory should be generated and it should contain the html you want to move to the Sphinx build's location.

Reply:
Yes, I checked. It worked. @ryanmrichard

Reply:
Okay. I must be missing something, regardless I'll go ahead and merge it then.